packages/components/inputs/selectable-search-input/README.md

```diff
@@ -95,8 +89,8 @@ export default Example;
 | `menuHorizontalConstraint` | `union`<br/>Possible values:<br/>`3 , 4 , 5`                                           |          | `3`       | Horizontal size limit for the dropdown menu.                                                                                                                                                                                                                                                                                                                                             |
 | `showSubmitButton`         | `boolean`                                                                              |          | `true`    | Show submit button in the input                                                                                                                                                                                                                                                                                                                                                          |
-| `selectDataProps`          | `Array: TDataProps[]`                                                                  |          |           | used to pass `data-*` props to the select component                                                                                                                                                                                                                                                                                                                                      |
-| `inputDataProps`           | `Array: TDataProps[]`                                                                  |          |           | used to pass `data-*` props to the input element                                                                                                                                                                                                                                                                                                                                         |
+| `selectDataProps`          | `Record`                                                                               |          |           | used to pass `data-*` props to the select component                                                                                                                                                                                                                                                                                                                                      |
```

Comment:
Not sure why this type has changed and the previous one is more accurate.
Also, I think it would help if we include an example of the expected object in the prop description.